packages/launchpad/src/signingcosmosclient.ts

```diff
@@ -14,13 +14,38 @@ export interface FeeTable {
   readonly send: StdFee;
 }
 
-const defaultFees: FeeTable = {
-  send: {
-    amount: coins(2000, "ucosm"),
-    gas: "80000", // 80k
-  },
+export interface FeeOptions {
+  readonly feeToken: string;
+  readonly gasPrice: number;
+}
```

Comment:
This can be changed to a `GasPrice` class with a `denom` and an `amount` (similar to coin, but decimal). Then we can create a `GasPrice.fromString()` parser that taked the same represenation as in the network config:

```toml
# The minimum gas prices a validator is willing to accept for processing a
# transaction. A transaction's fees must meet the minimum of any denomination
# specified in this config (e.g. 0.25token1;0.0001token2).
minimum-gas-prices = "0.025ushell"
```

We only need one denom per network.

Reply:
Do you know if there's a spec for that format somewhere?

Reply:
Hmm, no. The only thing I am aware of is this change: https://github.com/cosmos/cosmos-sdk/pull/7027. So we can use any `[a-z]` as the first character for the split.

Reply:
Once you have the split position, you can just use `Decimal.fromUserInput("0.025", 18)`, which is the Cosmos SDK precision for decimaps afaik.